admin-ui/app/types/react-grid-layout.d.ts

```diff
@@ -0,0 +1,90 @@
+declare module 'react-grid-layout' {
+  import { ComponentType } from 'react'
+
```

Comment:
Fix missing React namespace import

Line 6 references `React.CSSProperties`, but the React namespace is not imported. Only `ComponentType` is imported from 'react'. This will cause a TypeScript compilation error.



Apply this diff to import the React namespace:

```diff
-  import { ComponentType } from 'react'
+  import * as React from 'react'
+  import { ComponentType } from 'react'
```

Alternatively, use the imported types directly without the namespace:

```diff
   import { ComponentType } from 'react'
+  import type { CSSProperties, ReactNode } from 'react'

   interface GridLayoutProps {
     className?: string
-    style?: React.CSSProperties
+    style?: CSSProperties
     // ... other props
-    children?: React.ReactNode
+    children?: ReactNode
   }
```